src/stateless/icon.vue

```diff
@@ -226,6 +240,4 @@ export default {
 
 @keyframes SPIN {
     0%   { transform: rotate(0); }
-    100% { transform: rotate(720deg); }
-}
-</style>
+    100% { trans
```

Comment:
oops :)
Why is this not failing the tests btw? :smile:

Reply:
It compiled without errors, no idea why...